src/javascripts/ng-admin/Main/MainModule.js

```diff
@@ -40,6 +40,7 @@ define(function (require) {
 
     MainModule.directive('maDashboardPanel', require('ng-admin/Main/component/directive/maDashboardPanel'));
     MainModule.directive('menu', require('ng-admin/Main/component/directive/Menu'));
+    MainModule.directive('maJsonValidator', require('ng-admin/Main/component/directive/maJsonValidator'));
```

Comment:
I really don't knew where to put this file, is `Crud/field/maJsonValidator.js` sounds better for you ?